packages/dd-trace/src/config.js

```diff
@@ -13,85 +13,150 @@ class Config {
   constructor (options) {
     options = options || {}
 
-    const enabled = coalesce(options.enabled, platform.env('DD_TRACE_ENABLED'), true)
-    const profilingEnabled = coalesce(options.profiling, platform.env('DD_PROFILING_ENABLED'), false)
-    const debug = coalesce(options.debug, platform.env('DD_TRACE_DEBUG'), false)
-    const logInjection = coalesce(options.logInjection, platform.env('DD_LOGS_INJECTION'), false)
-    const url = coalesce(options.url, platform.env('DD_TRACE_AGENT_URL'), platform.env('DD_TRACE_URL'), null)
-    const site = coalesce(options.site, platform.env('DD_SITE'), 'datadoghq.com')
-    const hostname = coalesce(
+    this.tags = {}
+
+    // DD_TAGS
+    tagger.add(this.tags, platform.env('DD_TAGS'))
+    tagger.add(this.tags, platform.env('DD_TRACE_TAGS'))
+    tagger.add(this.tags, platform.env('DD_TRACE_GLOBAL_TAGS'))
+    tagger.add(this.tags, options.tags)
+
+    // DD_TRACE_ANALYTICS_ENABLED=false
```

Comment:
Why these comments?

Reply:
documenting that we're following a standardized set of environment variables and what their defaults should be i guess?

Reply:
The documentation is redundant being just above the actual code, and the default values are already documented in the API documentation and in the TypeScript type definitions.